lib/plugins/scoreboard.js

```diff
@@ -57,6 +58,47 @@ function inject (bot) {
     }
   })
 
+  bot.on('teams', (packet) => {
+    if (packet.mode === 0) {
+      const team = {}
+      team.displayName = packet.name
+      team.prefix = packet.prefix
+      team.suffix = packet.suffix
+      team.friendlyFire = packet.friendlyFire
+      team.nameTagVisibility = packet.nameTagVisibility
+      team.collisionRule = packet.collisionRule
+      team.color = packet.color
+      team.players = packet.players
+      teams[packet.team] = team
+    }
+
+    if (packet.mode === 1) {
+      delete teams[packet.team]
+    }
+
+    if (packet.mode === 2) {
+      if (teams[packet.team] === undefined) return
+      teams[packet.team].displayName = packet.name
+      teams[packet.team].prefix = packet.prefix
+      teams[packet.team].suffix = packet.suffix
+      teams[packet.team].friendlyFire = packet.friendlyFire
+      teams[packet.team].nameTagVisibility = packet.nameTagVisibility
+      teams[packet.team].collisionRule = packet.collisionRule
+      teams[packet.team].color = packet.color
```

Comment:
Are you sure that none of these values are `undefined`?

Reply:
none of these can be undefined, as they all have default values in the vanilla server/client

Reply:
Have you actually tested this?

Reply:
yes ive run it on my own bot on several non vanilla servers already, seems to work fine